src/w3c/headers.js

```diff
@@ -429,15 +429,15 @@ export function run(conf) {
         alt.hasOwnProperty("lang") && alt.lang ? ` hreflang='${alt.lang}'` : "";
       optional +=
         alt.hasOwnProperty("type") && alt.type ? ` type='${alt.type}'` : "";
-      return `<a rel='alternate' href='${alt.uri}'${optional}>${alt.label}</a>`;
+      return hyperHTML `<a rel='alternate' href='${alt.uri}'${optional}>${alt.label}</a>`;
     });
   if (conf.bugTracker) {
     if (conf.bugTracker.new && conf.bugTracker.open) {
-      conf.bugTrackerHTML = `<a href='${conf.bugTracker.new}'>${conf.l10n.file_a_bug}</a> ${conf.l10n.open_parens}<a href='${conf.bugTracker.open}'>${conf.l10n.open_bugs}</a>${conf.l10n.close_parens}`;
+      conf.bugTrackerHTML = hyperHTML `<a href='${conf.bugTracker.new}'>${conf.l10n.file_a_bug}</a> ${conf.l10n.open_parens}<a href='${conf.bugTracker.open}'>${conf.l10n.open_bugs}</a>${conf.l10n.close_parens}`;
```

Comment:
We should make sure that any current use of `[conf.bugTrackerHTML]` should be `conf.bugTrackerHTML` instead. (Because the former one is for raw strings)